api/src/services/users.js

```diff
@@ -457,18 +458,92 @@ const getUpdatedSettingsDoc = (username, data) => {
   });
 };
 
+const isDbAdmin = user => {
+  return couchSettings
+    .getCouchConfig('admins')
+    .then(admins => admins && !!admins[user.name]);
+};
+
+const saveUserUpdates = async (user) => {
+  const isAdmin = await isDbAdmin(user);
```

Comment:
Do we need to check if the user is an admin if we don't have a password field to change? I think we can have this small optimization so we don't call this endpoint unless necessary.

Reply:
Nice, done